Factory/SimpleFactory/AccountSimpleFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Factory.SimpleFactory
+{
+    class AccountSimpleFactory
+    {
+        public Account CreateBasicAccount(string name, int age)
```

Comment:
It shouldn't be different methods for account creation. And again no code with this pattern usage.

Reply:
It turns out that the simple factory is not a pattern, so I removed it.